bin/interpreter.sh

```diff
@@ -47,6 +47,24 @@ fi
 
 . "${bin}/common.sh"
 
+INTERPRETER_ID=$(basename "${INTERPRETER_DIR}")
+
+
+if [[ -z "${ZEPPELIN_INTP_MEM}" ]]; then
+  export ZEPPELIN_INTP_MEM="${ZEPPELIN_INTP_DEFAULT_MEM}"
+fi
+
+INTERPRETER_SPECIFIC_MEM_VARIABLE=`echo $INTERPRETER_ID | awk '{print toupper($0)"_INTP_MEM" }'`
```

Comment:
I think this makes more environments variables like FLINK_INTP_MEM, SPARK_INTP_MEM and so on. I don't think it's good way. It looks good to use the interpreter settings from web UI. What do you think of it?